libs/ui/components/navigation/breadcrumb/props.ts

```diff
@@ -0,0 +1,7 @@
+export interface BreadcrumbElement {
```

Comment:
with interfaces add `I` prefix
```suggestion
export interface IBreadcrumbElement {
```

Reply:
Todo

Reply:
And move it to types.ts